presentation/src/main/java/org/sagebionetworks/assessmentmodel/presentation/compose/TopBar.kt

```diff
@@ -17,12 +17,12 @@ import org.sagebionetworks.assessmentmodel.presentation.ui.theme.SageSurveyTheme
 fun PauseTopBar(
```

Comment:
So... is this only used for questions? (ie. not for instructions?) I am not seeing where you are flipping the value for instructions (which shouldn't show the skip button).

Reply:
Instructions use a CloseTopBar, which doesn't have a skip button.